### PR TITLE
Updated dot notation syntax to use dot-notation for all properties (in latest API)

### PR DESCRIPTION
**Pull request by capnslipp**
Updated the code sample for the **Dot Notation Syntax** section to use dot-notation syntax for all properties in the latest API.  As a result, changed some (correct) usage of dot-notation syntax in the _“Not:”_ code sample to use square-bracket syntax, for clarity.

The reason behind these code samples originally having square-bracket syntax for some calls in the _“For example:”_ sample and dot-notation syntax for those calls in the _“Not:”_ sample was that formerly, all class-level getters in Obj-C were just plain methods.  That changed in Xcode 8 _(and continues in the current iOS/macOS API as of Xcode 9.4.1 and the latest Xcode 10 beta)_ with the introduction of `class`-level properties in Objective-C, and the consequent updating of the iOS/macOS APIs by Apple.  Now both `UIColor`'s `orangeColor` and and `UIApplication`'s `sharedApplication` are true properties:

```objc
// From the macOS SDK documentation that ships with Xcode 8.4.1:

@property(class, strong, readonly) NSColor *orangeColor;

@property(class, nonatomic, readonly) UIApplication *sharedApplication;
```
While it could be understandable that NYTimes might want to keep calls to things like `[UIApplication sharedApplication]` using square-bracket syntax for consistency with their/your existing codebase, that causes a problem for potential new Obj-C programmers using your guide to help them learn Obj-C, since they'll read _“Dot notation is RECOMMENDED […] for getting […] properties”_ but then check the official Obj-C docs and see that `sharedApplication`, leading to confusion over the apparent non-sensical nature of your style guide.  It doesn't help that **Dot Notation Syntax** is the very first section in the style guide, and could turn off new programmers to the validity of your style guide _(despite the fact that a few years ago for Xcode ≤7, the style guide was accurate)_.

So in summary, I believe it makes sense to update code samples to use dot-notation syntax where Apple has updated the iOS/macOS APIs to use properties.  Even in an era of Swift, Obj-C is still receiving updates and consistency clean-up by Apple — _which is a wonderful thing for us long-time Obj-C devs_ — and so it makes sense to embrace that.

Changes made:

* In the **Dot Notation Syntax** section of each translated `README….md`, changed the _“For example:”_ code sample to use `UIColor.orangeColor` & `UIApplication.sharedApplication`, and changed the _“Not:”_ code sample to use `[UIColor orangeColor]` & `[UIApplication sharedApplication]`.

### CODE REVIEW

**Comment by capnslipp** [comment]
Forgot to mention this is the commit message: A nice rundown of the addition of `class` properties to Obj-C can be found at [Use Your Loaf's Objective-C Class Properties blog post](https://useyourloaf.com/blog/objective-c-class-properties/).

**Comment by loganmoseley** [comment]
Excellent suggestion and write-up. Dot-notation is indeed how I at least would write ObjC now. Thanks for contributing!

(Somebody else here will have to merge.)

**Comment by mattbischoff** [comment]
Great PR! I agree that this should be merged ASAP.

**Comment by Veronique** [comment]
Agreed.
